src/generator/src/domain/useCases.js

```diff
@@ -43,8 +43,10 @@ module.exports = async ({ generate, filesystem, options }) => async () => {
           request: await generateRequest(schema)
         }
       })
-      let type = 'mutation'
-      if (useCaseName.includes('find')) { type = 'query' }
+
+      let type = 'create'
+      if (useCaseName.includes('getById')) { type = 'read' } else if (useCaseName.includes('update')) { type = 'update' } else if (useCaseName.includes('delete')) { type = 'delete' }
```

Comment:
tip:
https://ultimatecourses.com/blog/deprecating-the-switch-statement-for-object-literals

Reply:
I had tied to implement switch before but this problems can't be resolved with switch case, but today I got an insight and implement it with a loop